Lib/functools.py

```diff
@@ -959,15 +959,12 @@ def __isabstractmethod__(self):
 ### cached_property() - computed once per instance, cached as attribute
```

Comment:
This line should also be changed. @carljm

Reply:
It's not an absolute guarantee in the presence of multi-threading race conditions, but this is still a generally accurate description of the usual behavior of a `cached_property`. What alternative wording would you suggest that is both clear and concise?

Reply:
How about 
```py
### cached_property() - property result cached as instance attribute
```

Reply:
Works for me! I've rolled that change into https://github.com/python/cpython/pull/106380 where I'm already touching `cached_property`.